### DISCUSSION TITLE
"Topics" page has changed?

### DISCUSSION THREAD

**Discussion by GavynBryan** [community/community#22959] [New to GitHub]
<p>Used to, when you go to a certain topic, you’d have the option of scrolling down and seeing more than 15 respositories matching the same topic. You could also sort by most recently updated/most popular/etc.</p>
<p>I don’t know what’s happened to the interface now…?</p>
<p>  </p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/1X/bcc3ac3975073bd3d7e6d6f97d61a21e5f035aa4.png" data-download-href="/uploads/short-url/qVSZsRYLy50mGpiTPRKxuPQeckI.png?dl=1" title="image.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/42544471/181074048-56b05e8e-11a7-4697-b480-29cea169fb2a.png" alt="image.png" data-base62-sha1="qVSZsRYLy50mGpiTPRKxuPQeckI" width="690" height="406" srcset="https://aws1.discourse-cdn.com/github/optimized/1X/bcc3ac3975073bd3d7e6d6f97d61a21e5f035aa4_2_690x406.png, https://aws1.discourse-cdn.com/github/optimized/1X/bcc3ac3975073bd3d7e6d6f97d61a21e5f035aa4_2_1035x609.png 1.5x, https://aws1.discourse-cdn.com/github/original/1X/bcc3ac3975073bd3d7e6d6f97d61a21e5f035aa4.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/1X/bcc3ac3975073bd3d7e6d6f97d61a21e5f035aa4_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">image.png</span><span class="informations">1291×761 42 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>It was like this before and after I logged in. Have the buttons moved or what?</p>

**Comment by techtickaran** [comment]
<p>this is sick, It was so useful to sort and search from the topic section.</p>

**Comment by ghost** [comment]
<p>what do you expect from microsoft </p>

**Comment by GavynBryan** [comment]
<p>No kidding? And there doesn’t seem to be any announcement of interface changes or anyone else talking about it… very strange. Last time I was on the topic page waas a little over a week before starting this thread and I distinctly remember being able to use the “Topics” page to not only browse through, but narrow down code repositories matching certain criteria. It was a nice way to explore issues and see what everyone else is doing.</p>

**Comment by DevCharly** [comment]
<p>Unfortunately the topics page has change from very useful to nearly useless:</p>
<ul>
<li>shows only 8 repos, which even include archived repos and long time inactive repos</li>
<li>no longer possible to browse all repos of a topic</li>
<li>no longer possible to filter by programing language</li>
<li>no sorting</li>
</ul>
<p>Hope that this functionallity will come back soon…</p>

**Comment by asyagelski** [comment]
<p><strong>THANK YOU!</strong> I was <em>really</em> hoping I wasn’t the only one. I absolutely <strong>HATE</strong> this change. I’ve found so many libraries that I’ve used in different projects now and then, and now we’re only supposed to look at ~10? Seriously? And they hardly update or change! It seems like it’s trying to “cater” to you, but why? We don’t need that functionality.</p>

**Comment by GavynBryan** [comment, marked as answer]
<p><strong>Update</strong></p>
<p>So, using the manual search function will give you the same results that browsing by topic used to give you. </p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/d/dbd17e1e49d61e171b3973d74d65f0e3c655c4ff.png" data-download-href="/uploads/short-url/vmBoV8nz2jvz9O4utMne6fGnDk3.png?dl=1" title="image.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/42544471/181074053-fabcc7fa-0d04-4080-8bb8-3a347d44c2c9.png" alt="image.png" data-base62-sha1="vmBoV8nz2jvz9O4utMne6fGnDk3" width="690" height="361" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/d/dbd17e1e49d61e171b3973d74d65f0e3c655c4ff_2_690x361.png, https://aws1.discourse-cdn.com/github/optimized/2X/d/dbd17e1e49d61e171b3973d74d65f0e3c655c4ff_2_1035x541.png 1.5x, https://aws1.discourse-cdn.com/github/original/2X/d/dbd17e1e49d61e171b3973d74d65f0e3c655c4ff.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/d/dbd17e1e49d61e171b3973d74d65f0e3c655c4ff_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">image.png</span><span class="informations">1280×670 42.5 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>So, you <em>can</em> have the full list of matching repositories but you need to enter it yourself in the search bar. I’m slightly disappointed by this change, but I can live with it. </p>

**Comment by DevCharly** [comment]
<p>Hmm, sorry, but this is more a workaround than a solotion for me, because the search finds far more repos than there are in a specific topic.</p>

**Comment by GavynBryan** [comment]
<p>Well, looks like the interface is officially back now. Not sure what that was about, but glad to see things are back to normal!</p>
